tests/sentry/migrations/test_0730_add_subscription_fk_to_incident.py

```diff
@@ -138,6 +138,10 @@ def test(self):
         assert self.incident.subscription is not None
         assert self.incident.subscription == self.query_subscription
 
+        self.query_subscription.delete()
+        self.incident.refresh_from_db()
+        assert self.incident.subscription is None
```

Comment:
should we also assert that the incident still exists or would that throw when trying to access the subscription?